lib/App.js

```diff
@@ -53,6 +55,31 @@ class App {
     this._git = new GitCommands(appPath);
   }
 
+  /* 
+  * Returns true if typescript is set to true in the app.json 
+  */
+  static usesTypeScript({ appPath }) {
+    return fs.existsSync(path.join(appPath, 'tsconfig.json'));
+  }
+
+  async convertFromTypeScript(appPath) {
+    Log(colors.green('✓ Typescript detected. Compiling...'));
+  
+    try {
+      const newPath = path.join(appPath, 'build/');
+      await fse.remove(newPath).catch(err => {});
+      await exec(`cd ${appPath} && npx -p typescript tsc --outDir build`);
```

Comment:
Ha I like using npx as a way to resolve the local tsc version. Kinda worried about what this is going to do if typescript isn't installed (I think it will always install and use the latest version which may or may not be what we want).

you can pass a `cwd` as option to `exec` to skip the `cd` see: https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback